test/autests/gold_tests/http2_abort/http2_abort.test.py

```diff
@@ -0,0 +1,98 @@
+'''
+Abort HTTP/2 connection.
+'''
+# @file
+#
+# Copyright 2022, Verizon Media
+# SPDX-License-Identifier: Apache-2.0
+#
+
+Test.Summary = '''
+Abort HTTP/2 connection.
+'''
+
+#
+# Test 1: Client abort after DATA frame
+#
+r = Test.AddTestRun("Client aborts after DATA frame")
+client = r.AddClientProcess("client1", "replay_files/client_after_data.yaml")
+server = r.AddServerProcess("server1", "replay_files/client_after_data.yaml")
+proxy = r.AddProxyProcess("proxy1", listen_port=client.Variables.https_port,
+                          server_port=server.Variables.https_port,
+                          use_ssl=True, use_http2_to_2=True)
+
+client.Streams.stdout += Testers.ContainsExpression(
+    'Client request aborts after DATA frame, with error code INTERNAL_ERROR.',
+    "Set client abort flag.")
+
+client.Streams.stdout += Testers.ContainsExpression(
+    'Testing client abort, submitting RST_STREAM frame for key 1 with error code INTERNAL_ERROR.',
+    "Detect client abort flag.")
+
+client.Streams.stdout += Testers.ContainsExpression(
+    'Sent frame: RST_STREAM',
+    "Send RST_STREAM frame.")
+
+proxy.Streams.stdout += Testers.ContainsExpression(
+    'Received RST_STREAM frame with error code INTERNAL_ERROR',
+    'Received RST_STREAM frame.'
+)
```

Comment:
Don't rely on the above Proxy Verifier messages to indicate the order. Instead, let's use `re.MULTILINE` with the data content before INTERNAL_ERROR to verify that the data was sent.

Reply:
This is in the proxy.

Reply:
Yes, the log is in the proxy. How do you know that the order is right between the frames though, given just the match with `Received RST_STREAM frame with error code INTERNAL_ERROR`?